Lets Do This/Controllers/Profile/LDTUserProfileViewController.m

```diff
@@ -115,18 +115,18 @@ - (void)styleView {
     self.headerView.backgroundColor = [UIColor colorWithPatternImage:[UIImage imageNamed:@"Header Background"]];
     [self styleBackBarButton];
 
-    self.nameLabel.text = [self.nameLabel.text uppercaseString];
-    [self.nameLabel setFont:[LDTTheme fontTitle]];
-    [self.nameLabel setTextColor:[UIColor whiteColor]];
+    self.nameLabel.text = self.nameLabel.text.uppercaseString;
+    [self.nameLabel setFont:LDTTheme.fontTitle];
```

Comment:
Hmm theres still more here (of course I catch after I merge. `self.nameLabel.font`